taigaProject/src/taigaApi/controller/project_controller.py

```diff
@@ -7,12 +8,12 @@
 
 router = APIRouter()
 
-@router.get("/Project")
-def auth(projectRequest: ProjectRequest):
-    project_info = get_project_by_slug(projectRequest.projectslug, projectRequest.authtoken)
+@router.post("/project")
```

Comment:
Keep it as Project, APIs should be in caps

Reply:
Updated. However, I found most of the website suggesting to use lowercase in the urls. Even Google search does that unless it's a query param.